srcpkgs/inkscape/template

```diff
@@ -35,4 +35,6 @@ post_patch() {
 		# since inkscape uses NULL instead of nullptr
 		vsed -e "/-Werror=format/d" -i CMakeScripts/DefineDependsandFlags.cmake
 	fi
+	# disable glyph tests that fail due to different hinting
+	vsed -e "s/\(test-glyph-y-pos\|test-rtl-vertical\|text-glyphs-combining\|text-glyphs-vertical\)/# \1 - skipped due to hinting issues/" -i testfiles/rendering_tests/CMakeLists.txt
```

Comment:
Can you split these into two or three lines? `vsed -e \` to make a multiline command.